pkg/gadgets/trace/exec/types/types.go

```diff
@@ -36,7 +36,8 @@ type Event struct {
 	UpperLayer bool     `json:"upperlayer" column:"upperlayer,width:10,fixed,hide"`
 	LoginUid   uint32   `json:"loginuid" column:"loginuid,template:uid,hide"`
 	SessionId  uint32   `json:"sessionid" column:"sessionid,minWidth:10,hide"`
-	Cwd        string   `json:"cwd,omitempty" column:"cwd,width:40" columnTags:"param:cwd"`
+	Cwd        string   `json:"cwd,omitempty" column:"cwd,width:40" columnTags:"param:paths"`
+	Exe        string   `json:"exe,omitempty" column:"exe,width:40" columnTags:"param:paths"`
```

Comment:
Good idea. I suggest ExePath because it is similar to sysdig's `proc.exepath`.

Sysdig has both exe and exepath:
```
proc.exe                      The first command line argument argv[0] (truncated after 4096 bytes) which is usually the 
                              executable name but it could be also a custom string, it depends on what the user 
                              specifies. [...] 
proc.exepath                  The full executable path of the process [...]
```